charts/ai-service/values.yaml

```diff
@@ -72,10 +72,11 @@ probes:
 pdb:
   maxUnavailable: 30%
 
-pvc:
```

Comment:
@abiodun0 @mfadler please check other values.yaml where I started to add
# -- helm docs

please, all new changes in values.yaml must have helmdocs.

you can run `./scripts/helm_docs.sh` to render and see you did correct.

check e.g. web-app chart, rootIngress section.

Reply:
done for pvc and artifactsCache